test-engineering/contract-tests/docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
     environment:
       # The configuration preset to use
       MERINO__ENV: "ci"
+      MERINO__REMOTE_SETTINGS__CRON_INTERVAL_SEC: 3
```

Comment:
This only controls how often the Remote Settings cron job triggers, we're doing a few tasks inside this cron job including resyncing with Remote Settings. So let's also add `MERINO__REMOTE_SETTINGS__RESYNC_INTERVAL_SEC: 1`, which means Merino will resync with RS if the last sync happened 1 second ago.

Reply:
See [here](https://mozilla-services.github.io/merino/ops.html#leaf-providers) for more details.

Reply:
Thanks for this!

change applied: https://github.com/mozilla-services/merino/pull/409/commits/bb23aaf4f7a578cf4631b62d9d141bc28f3b4a3b